src/middleware/externalRequests.js

```diff
@@ -53,21 +53,23 @@ const performRequests = (requests, ctx) => {
         // Assign any data received from the response to the assigned id in the context
         return {[requestDetails.id]: res.data}
       })
-      .catch(error => {
+      .catch(err => {
+        error = err
         logger.error(
-          `Failed Request Config ${JSON.stringify(error.config, null, 2)}`
+          `Failed Request Config ${JSON.stringify(err.config, null, 2)}`
         )
-        if (error.response) {
+
+        if (err.response) {
           throw new Error(
-            `Incorrect status code ${error.response.status}. ${error.response.data.message}`
+            `Incorrect status code ${err.response.status}. ${err.response.data.message}`
           )
-        } else if (error.request) {
+        } else if (err.request) {
           throw new Error(
-            `No response from lookup '${requestDetails.id}'. ${error.message}`
+            `No response from lookup '${requestDetails.id}'. ${err.message}`
           )
         } else {
           // Something happened in setting up the request that triggered an Error
-          throw new Error(`Unhandled Error: ${error.message}`)
+          throw new Error(`Unhandled Error: ${err.message}`)
```

Comment:
@bradsawadye This error assign here is quite weird, I understand why, but it's still hard to understand on first look. Perhaps making the variables more descriptive will help explain the code and because `err` isn't consistent with the rest of the caught error variables in the project.

```js
let responseTimestamp, orchestrationError, response

...

.catch(error => {
  orchestrationError = error
  ...
})
.finally(() => {
...
createOrchestration(
...,
error: orchestrationError,
...
)

```